src/game/Utils/Timer_Control.h

```diff
@@ -88,19 +88,19 @@ extern CUSTOMIZABLE_TIMER_CALLBACK gpCustomizableTimerCallback;
 
 #else
 
-#define   UPDATECOUNTER( c )
-#define   RESETCOUNTER( c )		( giTimerCounters[ c ] = giClockTimer )
-#define   COUNTERDONE( c )		( ( ( giClockTimer = GetJA2Clock() ) - giTimerCounters[ c ] ) >  giTimerIntervals[ c ] ) ? TRUE : FALSE
+#define UPDATECOUNTER( c )
+#define RESETCOUNTER( c )		( giTimerCounters[ c ] = giClockTimer )
+#define COUNTERDONE( c )		( ( ( giClockTimer = GetJA2Clock() ) - giTimerCounters[ c ] ) >  giTimerIntervals[ c ] ) ? TRUE : FALSE
```

Comment:
not really whitespace, but brackets around the complete term are missing.

should be split into at least 2 lines

Reply:
I've missed this one

Reply:
And as we've neatly demonstrated some time ago: `#define`s with missing brackets can be really dangerous.